getting-started-tutorial/tutorial-1-MLRun-basics.ipynb

```diff
@@ -0,0 +1,1890 @@
+{
```

Comment:
<p>If every param is in a new line, why the <code>schedule</code> which is the only thing you really want to show here is in the same line as the inputs? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/demos/pull/73/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='77'/>